src/graph/graph.h

```diff
@@ -133,6 +133,15 @@ void Graph_LabelNode
 	uint label_count
 );
 
+// Remove assocoation of node with labels by removing the value 1 from label matrix L at position [id,id]
```

Comment:
add the remove of the entry in node label matrix NLM[node_id, label_id]
update also in line 127